code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -93,6 +93,44 @@
 			update_flags |= M.adjustToxLoss(1, FALSE)
 	return list(effect, update_flags)
 
+/datum/reagent/medicine/synaptizine/recal
+	name = "Recal"
+	id = "recal"
+	description = "An oily insulating liquid that passively regulates electrical activity on sensitive electronic components, allowing them to recover from decalibrating events faster. \
+	Overdosing will cause under-voltage errors and hamper component heat dissipation, potentially causing heat damage."
+	reagent_state = LIQUID
+	color = "#85845d"
+	overdose_threshold = 40
+	harmless = FALSE
+	taste_description = "mineral oil and toothpaste"
+	process_flags = SYNTHETIC
+
+/datum/reagent/medicine/synaptizine/recal/overdose_process(mob/living/M, severity)
+	var/list/overdose_info = ..()
+	var/effect = overdose_info[REAGENT_OVERDOSE_EFFECT]
+	var/update_flags = overdose_info[REAGENT_OVERDOSE_FLAGS]
+	if(severity == 1)
+		if(effect <= 1)
+			M.visible_message("<span class='warning'>[M] suddenly jitters for a moment.</span>")
+			M.AdjustJitter(2 SECONDS)
+		else if(effect <= 3)
+			M.emote("shudder")
+		if(effect <= 8)
+			update_flags |= M.adjustFireLoss(1, FALSE)
+			to_chat(M, "<span class='warning'>Your internals start to overheat!</span>")
+	else if(severity == 2)
+		if(effect <= 2)
+			M.visible_message("<span class='warning'>[M] suddenly jitters for a moment.</span>")
+			M.AdjustJitter(2 SECONDS)
+		else if(effect <= 5)
+			M.visible_message("<span class='warning'>[M] staggers and seizes up!</span>")
+			M.Dizzy(16 SECONDS)
+			M.Weaken(8 SECONDS)
+		if(effect <= 15)
+			update_flags |= M.adjustFireLoss(1, FALSE)
+			to_chat(M, "<span class='warning'>Your internals start to overheat!</span>")
+	return list(effect, update_flags)
```

Comment:
not a very big fan of this big block of copy pasted code with minimal changes trying to think of a way to make this better though.

you could make a proc called something like "overdose_visuals" and let it take the severity and the effect as a parameter and have that proc do the visible messages? 

overriding it this way almost insures that if synap is changed in the future this will likely be forgotten about but if you dont override it 100% then if synap is changed this will be changed as well.

Reply:
I've combined the effects into a single overdose proc so if someone comes in later and adjusts one, it'll be obvious that they'll have to adjust the other too.